models/src_dynamics_365_crm.yml

```diff
@@ -0,0 +1,39 @@
+version: 2
+
+sources:
+  - name: dynamics_365_crm 
+    database: "{% if target.type not in ('spark') %}{{ var('dynamics_365_crm_database', target.database) }}{% endif %}"
+    schema: "{{ var('dynamics_365_crm_schema', 'dynamics_365')}}"
+
+    loader: fivetran
+    loaded_at_field: _fivetran_synced
+      
+    freshness:
+      warn_after: {count: 72, period: hour}
+      error_after: {count: 96, period: hour}
```

Comment:
Let's remove the freshness here. I don't believe we know enough about the expected freshness of each source and I wouldn't want to impose tests when we aren't 100% sure. Also, we've run into issues in the past with source freshness tests, so it's best to ignore this.

Reply:
Good point. Removed.